### DISCUSSION TITLE
Risks of running React 16.2 with Emotion 10

### DISCUSSION THREAD

**Discussion by postrad** [emotion-js/emotion#2517] [Q&A] [answered]
I just discovered on a project that we've been using Emotion 10 for a while, but we are still on React 16.2. Are there any known risks to this? It seems to be working fine. Thanks!

**Comment by Andarist** [comment, marked as answer]
Are you sure that you are actually using React 16.2? Maybe your deps are using something like `^16.2.0` but the actual version that is installed is a more fresh one?

**Reply by postrad** [reply]
I'm certain that this project is still on v16.2, much to my chagrin. 😅

**Reply by Andarist** [reply]
I'm unsure how this can work - Emotion 10 was relying on `React.createContext` and this has only been introduced in [React 16.3](https://reactjs.org/blog/2018/03/29/react-v-16-3.html).

**Reply by postrad** [reply]
How strange! We have a pretty dense monorepo, but I just did a deep dive to see if somehow a newer version of React had somehow gotten introduced. But we are definitely on v16.2. I appreciate the answer, though, and if I learn anything useful I will happily share it here.

**Reply by postrad** [reply]
I'm sorry- it turns out there isn't any mystery at all. We were on React 16.12, not 16.2. I apologize for making such an obvious mistake.
